projects/observability/src/shared/components/entity-renderer/entity-renderer.component.ts

```diff
@@ -14,7 +14,7 @@ import { EntityNavigationService } from '../../services/navigation/entity/entity
       *ngIf="this.name"
       class="ht-entity-renderer"
       [ngClass]="{ 'default-text-style': !this.inheritTextStyle }"
-      [htTooltip]="this.name"
+      [htTooltip]="this.getTooltipText | htMemoize: this.name"
```

Comment:
```suggestion
      [htTooltip]="this.getTooltipText | htDisplayString : 'unknown'"
```
`htDisplayString` should be taking care of `null` and `'null'` values.

Reply:
Oh, forgot about this. Thanks!
Updated.